tests/test-cases/include-local-wildcard/integration.include-local.test.ts

```diff
@@ -1,21 +1,26 @@
 import {WriteStreamsMock} from "../../../src/write-streams";
 import {handler} from "../../../src/handler";
 import {initSpawnSpy} from "../../mocks/utils.mock";
+import assert, {AssertionError} from "assert";
 import {WhenStatics} from "../../mocks/when-statics";
+import chalk from "chalk";
 
 beforeAll(() => {
     initSpawnSpy(WhenStatics.all);
 });
 
 test("include-local-wildcard <build-job>", async () => {
```

Comment:
not sure if i am missing out something or there's some update to the gitlab, from gitlab pipeline editor,
```
---
include:
  local: '*.gitlab-ci.yml'
```
is invalid with the following error:
`This GitLab CI configuration is invalid: Maximum of 150 nested includes are allowed!.`
![image](https://github.com/firecow/gitlab-ci-local/assets/28011783/c201e031-0792-4270-9a62-b787ebbdc5e0)